test/base.js

```diff
@@ -0,0 +1,42 @@
+import { expect } from 'chai';
+import Base from '../base.js';
+
+describe('Base', () => {
```

Comment:
the set method is not tested and I think the set did not really change the value in attributes (never affected)

Reply:
the set method ? in `base.js` ? there is no such method